apps/openchallenges/organization-service/src/test/java/org/sagebionetworks/openchallenges/organization/service/exception/SimpleChallengeGlobalExceptionTest.java

```diff
@@ -0,0 +1,62 @@
+package org.sagebionetworks.openchallenges.organization.service.exception;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+
+public class SimpleChallengeGlobalExceptionTest {
+
+  @Test
+  public void SimpleChallengeGlobalException_ShouldReturnMessage_WhenMessageKeyIsPassed() {
+    // Create an instance of SimpleChallengeGlobalException using the constructor with details
+    String details = "Something went wrong";
+    SimpleChallengeGlobalException exception = new SimpleChallengeGlobalException(details);
+
+    // Verify the exception details
+
```

Comment:
This test must assert the `exception`

Reply:
Added assertion to check the message value